tools-local/procdump/Eula.txt

```diff
@@ -0,0 +1,75 @@
+﻿Sysinternals Software License Terms
+These license terms are an agreement between Sysinternals (a wholly owned subsidiary of Microsoft Corporation) and you. Please read them. They apply to the software you are downloading from technet.microsoft.com/sysinternals, which includes the media on which you received it, if any. The terms also apply to any Sysinternals
+* updates,
+* supplements,
+* Internet-based services,
+* and support services
+for this software, unless other terms accompany those items. If so, those terms apply.
+BY USING THE SOFTWARE, YOU ACCEPT THESE TERMS. IF YOU DO NOT ACCEPT THEM, DO NOT USE THE SOFTWARE.
```

Comment:
I don't think put binary in our source code is a good idea

Reply:
You may want to put the data in this repo - https://github.com/dotnet/machinelearning-testdata